### DISCUSSION TITLE
Scroll to top of div

### DISCUSSION THREAD

**Discussion by aksld** [alpinejs/alpine#2813] [1. Help] [answered]
Hello,

This is my codepen : https://codepen.io/aksld/pen/JjMpwYJ

In the red section, if you click on a picto, you see Lorem text in full screen height. I'm trying to ensure that when clicked, I scroll to the top of the red div. Can I do this with alpine functions ? Or I need to use native JS ?

Thanks for tips :)

**Comment by SimoTod** [comment, marked as answer]
`@click="expertise = 'title1'; $root.scrollIntoView({ behavior: 'smooth' })"`

**Reply by aksld** [reply]
Thanks :)

**Reply by judevs** [reply]
@SimoTod does this work well in safari? as far as I know `scrollIntoView()` does not work in safari properly 
![image](https://user-images.githubusercontent.com/32464575/205110255-ddaa63af-2c29-432b-a9c0-c3bbb0fa69f4.png)

If it doesn't work well in safari ios, is there an alternative solution to this?

**Reply by SimoTod** [reply]
It doesn't do smooth scrolling (just snap) but it's still scrolling to the right point (graceful degradation).
